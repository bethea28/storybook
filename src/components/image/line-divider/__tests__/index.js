/* @flow */
import React from 'react';
import { shallow } from 'enzyme';

import LineDivider from '..';

it('renders correctly', () => {
  const line = shallow(<LineDivider />);
  expect(line).toMatchSnapshot();
});
