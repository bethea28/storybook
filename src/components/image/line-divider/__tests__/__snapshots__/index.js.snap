// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "height": 5,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <View
    key="#e9e9e91"
    style={
      Object {
        "backgroundColor": "#e9e9e9",
        "flex": 1,
      }
    }
  />
  <View
    key="#f5f6f65"
    style={
      Object {
        "backgroundColor": "#f5f6f6",
        "flex": 5,
      }
    }
  />
</View>
`;
